<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MyWidget</class>
 <widget class="QWidget" name="MyWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MyWidget</string>
  </property>
  <widget class="QWidget" name="formLayoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>261</width>
     <height>121</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <property name="fieldGrowthPolicy">
     <enum>QFormLayout::ExpandingFieldsGrow</enum>
    </property>
    <property name="labelAlignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
    <item row="0" column="0">
     <widget class="QLabel" name="nLabel">
      <property name="text">
       <string>姓名(&amp;N)</string>
      </property>
      <property name="buddy">
       <cstring>nLineEdit</cstring>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="nLineEdit"/>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="sLabel">
      <property name="text">
       <string>性别(&amp;S)</string>
      </property>
      <property name="buddy">
       <cstring>sComboBox</cstring>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QComboBox" name="sComboBox"/>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="aLabel">
      <property name="text">
       <string>年龄(&amp;A)</string>
      </property>
      <property name="buddy">
       <cstring>aSpinBox</cstring>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QSpinBox" name="aSpinBox"/>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="mLabel">
      <property name="text">
       <string>邮箱地址(&amp;M)</string>
      </property>
      <property name="buddy">
       <cstring>mLineEdit</cstring>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QLineEdit" name="mLineEdit"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>226</y>
     <width>275</width>
     <height>25</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout" stretch="2,1">
    <property name="spacing">
     <number>16</number>
    </property>
    <property name="sizeConstraint">
     <enum>QLayout::SetDefaultConstraint</enum>
    </property>
    <item>
     <widget class="QPushButton" name="pushButton">
      <property name="text">
       <string>显示可扩展窗口</string>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QFontComboBox" name="fontComboBox"/>
    </item>
   </layout>
  </widget>
  <widget class="QTextEdit" name="textEdit">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>257</y>
     <width>382</width>
     <height>35</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>11</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
